app/Http/Controllers/Api/UsersController.php

```diff
@@ -74,6 +83,17 @@ public function store(UserRequest $request)
         }
 
         $contest->waitingRoom->users()->attach($user->id);
+        $this->manager->appendCampaign($contest);
+
+        // Fire off welcome Email
+        $message = Message::where(['contest_id' => $contest->id, 'type' => 'welcome'])->first();
```

Comment:
OH YOU MEAN THE ENTIRE oh

i thought you were just referring too the Message:where line